extras/wait-for-bricks.sh

```diff
@@ -0,0 +1,159 @@
+#!/bin/bash
+#Copyright (c) 2018 Red Hat, Inc. <http://www.redhat.com>
+#This file is part of GlusterFS.
+#
+#This file is licensed to you under your choice of the GNU Lesser
+#General Public License, version 3 or any later version (LGPLv3 or
+#later), or the GNU General Public License, version 2 (GPLv2), in all
+#cases as published by the Free Software Foundation.
+
+
+function volinfo_field()
+{
+    local vol=$1;
+    local field=$2;
+
+    gluster volume info $vol --xml| grep -oPm1 "(?<=<$field>)[^<]+"
```

Comment:
May be Missing space before |

Reply:
Please run this script through `shellcheck`, there are a number of things that the tool suggests to improve.